data/monitoring/prom-rule.yaml

```diff
@@ -11,6 +11,9 @@ spec:
       rules:
         - expr: sum(up{namespace='{{ .Namespace }}', pod=~'cluster-network-addons-operator-.*'} or vector(0))
           record: kubevirt_cnao_num_up_operators
+          annotations:
+            summary: Total count of running CNAO operators
+            type: Gauge
```

Comment:
If I understand correctly, this is a recording rule and not really a gauge.
If you look at `pkg/monitoring/metrics.go`
You'll we have two gauges there:
- kubevirt_cnao_cr_ready
- kubevirt_cnao_cr_kubemacpool_deployed
For that reason, I don't think we should label these records as gauges.
IIUC, these are metrics, not gauges.